conf/log4j.properties

```diff
@@ -12,63 +12,73 @@
 # You should have received a copy of the GNU General Public License along with this program.
 # If not, see <https://www.gnu.org/licenses/>.
 # ***** END LICENSE BLOCK *****
+status = error
+dest = err
+name = PropertiesConfig
+packages = com.zimbra.qa.selenium
 
 
 # Set root logger level to INFO and its only appender to A1.
-log4j.rootLogger=INFO, A1, general
 
 # A1 is set to be a ConsoleAppender.
-log4j.appender.A1=org.apache.log4j.ConsoleAppender
+appender.A1.type=org.apache.log4j.ConsoleAppender
 
 # A1 uses PatternLayout.
-log4j.appender.A1.layout=org.apache.log4j.PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+appender.A1.layout.type=PatternLayout
+appender.A1.layout.pattern=%-4r [%t] %-5p %c %x - %m%n
 
 # A2 is set to be a FileAppender.
-log4j.appender.A2=org.apache.log4j.RollingFileAppender
-log4j.appender.A2.File=${outputDirectory}/selenium.txt
-log4j.appender.A2.MaxFileSize=100KB
+appender.A2.type=org.apache.log4j.RollingFileAppender
+appender.A2.fileName=${outputDirectory}/selenium.txt
+appender.A2.policies.type=Policies
+appender.A2.policies.size.type=SizeBasedTriggeringPolicy
+appender.A2.policies.size.size=100KB
 
 # A2 uses PatternLayout.
-log4j.appender.A2.layout=org.apache.log4j.PatternLayout
-log4j.appender.A2.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+appender.A2.layout.type=PatternLayout
+appender.A2.layout.pattern=%-4r [%t] %-5p %c %x - %m%n
 
 # Test Case Step tracer
-#log4j.logger.testcase.trace=TRACE, A3
-#log4j.logger.com.zimbra.qa.selenium.framework.util.HarnessException=A3
+log4j.logger.testcase.trace=TRACE, A3
```

Comment:
To define a logger below are the steps :
logger.{name}.name = fully classified class name/ package name/ category name/ unique name
logger.{name}.level = info/warn etc
logger.{name}.additivity = false
logger.{name}.appenderRef.{refname}.ref = {appenderRef}